test/integration/manifests/cilium/cilium-config-hubble.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
```

Comment:
where you are deploying this cilium config? I didn't see this config getting deployed in cilium-e2e-step-template